internal/common/baseclient/baseclient.go

```diff
@@ -131,18 +130,16 @@ func (c *BaseClient) GetVoteForRule(ruleName string) bool {
 
 // GetVoteForElection returns the client's Borda vote for the role to be elected.
 // COMPULSORY: use opinion formation to decide a rank for islands for the role
-func (c *BaseClient) GetVoteForElection(roleToElect shared.Role) []shared.ClientID {
+func (c *BaseClient) VoteForElection(roleToElect shared.Role, candidateList []shared.ClientID) []shared.ClientID {
 	// Done ;)
 	// Get all alive islands
```

Comment:
Which bit is doing that? I don't see code near here extracting alive clients

Reply:
Oh, I'm sorry I forget to delete that comment line.